packages/x-charts/src/internals/animation/useAnimate.test.tsx

```diff
@@ -103,7 +103,7 @@ describe('useAnimate', () => {
     const { rerender } = render(<TestComponent width={2000} />);
 
     await waitNextFrame();
-    expect(callCount()).to.be.equal(2);
+    expect(callCount()).to.be.equal(reactMajor > 18 ? 3 : 2);
```

Comment:
@LukasTy do you know the reasoning behind this? 

Why did this change suddenly became needed after updating our internal packages? 🤔

Reply:
To be honest, I haven't thoroughly investigated the root cause. 🙈 
Maybe it's because of https://github.com/mui/material-ui/pull/45812? 🤔 
cc @bernardobelchior